pages*: add missing mnemonics (#12151)
tldr-pages/tldr: pages/android/am.md

> Android activity manager.
> More information: <https://developer.android.com/studio/command-line/adb#am>.

> - Start an activity with the specific component and package [n]ames:

Reviewer: There's only one name in this example.
```suggestion
- Start an activity with a specific component and package [n]ame:
```
